src/actions/index.js

```diff
@@ -15,7 +15,7 @@ export const setStudent= student => ({
 })
 
 export const setLessons= lessons => ({
-  type: 'SET_STUDENT',
+  type: 'SET_LESSONS',
   lessons
```

Comment:
This took me a hot second to figure out - but once I did, lessons were loaded immediately!